Utilities/config.json

```diff
@@ -1 +1,3 @@
-{"version":1,"swiftSyntaxVersionForMacroTemplate":{"major":600,"minor":0,"patch":0, "prereleaseIdentifier":"latest"}}
+{"version":1,
+  "swiftSyntaxVersionForMacroTemplate":{"major":600,"minor":0,"patch":0, "prereleaseIdentifier":"latest"},
+  "swiftTestingVersionForTestTemplate":{"major":0,"minor":7,"patch":0}}
```

Comment:
We recently released tag 0.8.0 so you can bump this, as well as another place I saw in this PR

Reply:
I'll bump in both places, thanks!